it/src/test/scala/com/wavesplatform/it/sync/matcher/OrderBookTestSuite.scala

```diff
@@ -0,0 +1,75 @@
+package com.wavesplatform.it.sync.matcher
+
+import com.typesafe.config.Config
+import com.wavesplatform.it.ReportingTestName
+import com.wavesplatform.it.api.SyncHttpApi._
+import com.wavesplatform.it.api.SyncMatcherHttpApi._
+import com.wavesplatform.it.sync.matcher.config.MatcherDefaultConfig._
```

Comment:
Use MatcherPiceAssetConfig. In test jsut issue assets. All pairs already created in config